frontend/pages/admin/OrgSettingsPage/cards/Advanced/Advanced.tsx

```diff
@@ -38,17 +47,39 @@ const Advanced = ({
     enableStartTLS,
     enableHostExpiry,
     hostExpiryWindow,
+    deleteActivities,
+    activityExpiryWindow,
     disableLiveQuery,
     disableScripts,
     disableQueryReports,
   } = formData;
 
   const [formErrors, setFormErrors] = useState<IAppConfigFormErrors>({});
 
+  const activityExpiryWindowOptions = useMemo(
+    () =>
+      getCustomDropdownOptions(
+        ACTIVITY_EXPIRY_WINDOW,
```

Comment:
+ `__OPTIONS` suffix, and maybe even `DEFAULT__` prefix, would make this more clear (same as in constants file)